IPython/terminal/pt_inputhooks/asyncio.py

```diff
@@ -28,13 +28,30 @@
 
 """
 import asyncio
+from prompt_toolkit import __version__ as ptk_version
+
+PTK3 = ptk_version.startswith('3.')
```

Comment:
This will result in incorrect logic when pt4 comes out. Let's parse it out correctly:

```suggestion
PTK3 = int(ptk_version.split('.', 1)[0]) >= 3
```

Reply:
My guess is PTK4 will also have a different API; so wether we go through the PTK2 or PTK3 branch it will fail.

It's also the same check we have in other locations... But up to @jonathanslenders

Reply:
I don't really have any opinion. In setup.py, we explicitly mention to not install PTK4, and at the point where we upgrade, it's easy enough to fix this to whatever makes sense.